api/openapi.yaml

```diff
@@ -901,17 +933,47 @@ components:
                 type: number
                 format: float
               waypoints:
-                type: array
-                items:
-                  type: object
-                  properties:
-                    track_section:
-                      type: integer
-                    geo_coordinate:
-                      type: array
-                      items: { type: number, format: float }
-                      minItems: 2
-                      maxItems: 2
+                $ref: "#/components/schemas/Waypoint"
+    Waypoint:
+      type: array
+      items:
+        type: object
+        properties:
+          track_section:
+            type: integer
+          geo_coordinate:
+            type: array
+            items: { type: number, format: float }
+            minItems: 2
+            maxItems: 2
+    STDCMRequest:
+      properties:
+        infra:
+          type: integer
+        rolling_stocks:
+          type: array
+          minItems: 1
+          items:
+            type: integer
+        timetable:
+          type: integer
+          description: timetable ID
+        start_time:
+          type: number
+          format: double
+        end_time:
+          type: number
+          format: double
```

Comment:
Why do we have both a start and end time?

Reply:
Eventually we'll be able to specify either one as input